store/src/main/java/com/dropbox/android/external/store4/StoreRequest.kt

```diff
@@ -63,7 +70,8 @@ data class StoreRequest<Key> private constructor(
         fun <Key> cached(key: Key, refresh: Boolean) = StoreRequest(
             key = key,
             skippedCaches = 0,
-            refresh = refresh
+            refresh = refresh,
+            fallbackToSourceOfTruth = true
```

Comment:
I think this is a breaking change. Maybe default to false and make configurable

Reply:
This parameter doesn't really have any meaning for cached requests because disk will already be unlocked anyway, so I assumed that conceptually it's true by default. And all the tests pass.

I know it's a bit redundant, but since we cannot distinguish between request types (all the request parameters are shared amongst all types), I was not sure how to represent this behavior only for fresh.

Should we go with a different API? That `tryFresh` you mentioned on https://github.com/dropbox/Store/issues/394#issuecomment-1024183802 would have the same problem.